llvm/test/tools/llvm-objcopy/MachO/install-name-tool.test

```diff
@@ -0,0 +1,26 @@
+## This test checks general llvm-install-name-tool behavior
+
+# RUN: yaml2obj %s -o %t
+
+## Passing a non-Mach-O binary
+# RUN: not llvm-install-name-tool -add_rpath foo %t 2>&1 | FileCheck %s --check-prefix=NON_MACH_O
```

Comment:
I forget, does this invocation just normally replace `%t` inline, or is an output file created? If an output file could be created, it might be worth validating that the file doesn't exist afterwards (you should make sure to delete it beforehand, in case one is left over from a previous run).

Reply:
this tool overwrites the passed output, so it has to exist before and still will after